docs/cardano/liquid-staking/c1-overview/contract-overview.mdx

```diff
@@ -65,8 +65,6 @@ Since rewards are automatically compounding, you do not need to call this functi
 
 ### 3. ERC20 interface
 
-*sstMilkADA* is an ERC20 token
```

Comment:
Just deleting `sstMilkADA` definitely isn't right because as we've discussed there are two contracts: an ADA-denominated contract and a share-denominated contract